### PR TITLE
chore(docs): Update Fx 120 release notes for release cycle

### PR DESCRIPTION
**Pull request by bsmth**
Release note work for Fx 120.

__changes:__

* Clean up empty sections in 120 relnotes
* "Nightly" -> "Beta"
* Add template for 122

__Note:__
The [RapidRelease calendar at wiki.mozilla.org](https://wiki.mozilla.org/RapidRelease/Calendar#Future_branch_dates) has a deprecation notice to use https://whattrainisitnow.com instead. I am switching this URL out in the current docs and in the template.

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/121/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ page-type: firefox-release-notes
 
 {{FirefoxSidebar}}
 
-This article provides information about the changes in Firefox 121 that affect developers. Firefox 121 is the current [Nightly version of Firefox](https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly) and ships on [December 19, 2023](https://wiki.mozilla.org/RapidRelease/Calendar#Future_branch_dates).
+This article provides information about the changes in Firefox 121 that affect developers. Firefox 121 is the current [Beta version of Firefox](https://www.mozilla.org/en-US/firefox/channel/desktop/#beta) and ships on [December 19, 2023](https://whattrainisitnow.com/calendar/).
```
Comment (zfox23) [inline files/en-us/mozilla/firefox/releases/121/index.md:9]: similar to the comment above. wdyt?

```suggestion
This article provides information about the changes in Firefox 121 that affect developers. Firefox 121 is the current [Beta version of Firefox](https://www.mozilla.org/en-US/firefox/channel/desktop/#beta) and ships on [December 19, 2023](https://whattrainisitnow.com/release/?version=121).
```

File: files/en-us/mozilla/firefox/releases/122/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: Firefox 122 for developers
+slug: Mozilla/Firefox/Releases/122
+page-type: firefox-release-notes
+---
+
+{{FirefoxSidebar}}
+
+This article provides information about the changes in Firefox 122 that affect developers. Firefox 122 is the current [Nightly version of Firefox](https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly) and ships on [January 23, 2024](https://whattrainisitnow.com/calendar/).
```
Comment (zfox23) [inline files/en-us/mozilla/firefox/releases/122/index.md:9]: Similar comment to above; wdyt?

```suggestion
This article provides information about the changes in Firefox 122 that affect developers. Firefox 122 is the current [Nightly version of Firefox](https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly) and ships on [January 23, 2024](https://whattrainisitnow.com/release/?version=122).
```

**Review by zfox23 — CHANGES_REQUESTED** [review]
changing links; what do you think?

**Comment by bsmth** [comment]
> changing links; what do you think?

Good call, suggestions applied in https://github.com/mdn/content/pull/30434/commits/0b06b78301a18b310f0e4509c2de12ada2271f85

**Review by zfox23 — APPROVED** [review]

**Pull request merged**